torch/_inductor/select_algorithm.py

```diff
@@ -1110,7 +1126,10 @@ def log_results(
         autotune_type_str = (
             "SubProcess" if config.autotune_in_subproc else "SingleProcess"
         )
-        sys.stderr.write(f"{autotune_type_str} AUTOTUNE takes {elapse:.4f} seconds\n")
+        sys.stderr.write(
```

Comment:
Why don't we use logger here and write to stderr? hmm..

Reply:
I guess the code is written before we have TORCH_LOGS and the pure logging.Logger is not easy to turn on/off from command line so there is less incentive to do that.